versions.json

```diff
@@ -32,10 +32,6 @@
       "version": "8.16.5",
       "branch": "8.16",
       "previousMajor": true
-    },
-    {
-      "version": "7.17.28",
```

Comment:
We'll likely want to keep this until 9.0 is released.

Reply:
yeah, re-added the bump as usually.